test/integration/components/droplist.js

```diff
@@ -165,4 +165,58 @@ context('Droplist', function() {
       .get('.picklist')
       .should('not.exist');
   });
+
+  specify('isCheckable', function() {
+    let selectlist;
+    const Droplist = this.Droplist;
+
+    cy
+      .getHook($hook => {
+        const $div = $hook.append('<div style="position: absolute; bottom: 20px; right: 20px;"></div>');
+
+        const region = new Region({
+          el: $div[0],
+        });
+
+        selectlist = new Droplist({
+          picklistOptions: {
+            headingText: 'Test Options',
+            isCheckable: true,
+          },
+          collection,
+        });
+
+        selectlist.showIn(region);
+      });
+
+    cy
+      .get('@hook')
+      .contains('Choose One...')
+      .click();
+
+    cy
+      .get('.picklist')
+      .find('.js-picklist-item')
+      .first()
+      .click();
+
+    cy
+      .get('@hook')
+      .contains('Option 1')
+      .click();
+
+    cy
+      .get('.picklist')
+      .find('.js-picklist-item')
+      .first()
+      .find('.icon')
+      .should('have.class', 'fa-circle-check');
+
+    cy
+      .get('.picklist')
+      .find('.js-picklist-item')
+      .last()
+      .find('.icon')
+      .should('not.exist');
+  });
 });
```

Comment:
Simple test that:

1. Opens the droplist.
2. Selects a picklist item in the droplist and verifies the check mark icon is shown for it.
3. Verifies only the one picklist item has the check mark. This ensures it's not applied to multiple picklist items and only the selected one. Possibly overkill.